packages/smooth_app/lib/helpers/launch_url_helper.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class LaunchUrlHelper {
```

Comment:
If `LauncherUrlHelper` is only going to be used a namespace for helper functions, it might be better to make it such with something like:

```dart
class LaunchUrlHelper {
  LaunchUrlHelper._();

  static Future<void> launchURL(String url, bool isOFF) async {
    ...
  }
}
```

which makes it clear that this isn't meant to be instantiated (private constructor).